pmm/v3/pmm3-client-autobuild-amd.groovy

```diff
@@ -133,11 +138,18 @@ pipeline {
                                 """
                             }
                         }
+                        stage('Build client binary rpm AL2023') {
+                            steps {
+                                sh """
```

Comment:
```suggestion
                                sh '''
```
The best practice, according to Jenkins' manuals, is to use single quotes everywhere.